framework/math/quadratures/angular/legendre_poly/assoc_legendrepoly.cc

```diff
@@ -4,14 +4,15 @@
 #include "framework/math/quadratures/angular/legendre_poly/legendrepoly.h"
 #include <cmath>
 #include <cstdlib>
+#include <utility>
 
 namespace opensn
 {
 
 double
 AssocLegendre(unsigned int ell, int m, double x)
 {
-  if (abs(m) > ell)
+  if (std::cmp_greater(abs(m), ell))
```

Comment:
Even though cmp_greater is straightforward, what's the push to remove >? Are the variables different flavors of integers?

Reply:
Can we resolve this @ragusa ?

Reply:
Ok. I added `ell` on the list of thing to fix in #90. So resolving this discussion and merging PR.